paddle/fluid/inference/tests/api/tester_helper.h

```diff
@@ -183,7 +177,8 @@ void SetFakeImageInput(std::vector<std::vector<PaddleTensor>> *inputs,
     float *input_data = static_cast<float *>(input.data.data());
     // fill input data, for profile easily, do not use random data here.
     for (size_t j = 0; j < len; ++j) {
-      *(input_data + j) = Random(0.0, 1.0) / 10.;
+      *(input_data + j) =
+          static_cast<float>((j + continuous_inuput_index) % len) / len;
```

Comment:
To simulate random vector in (0,1) ?

Reply:
Yeah. Different input are generated each time,  but it's fixed on the whole. There will be no random situation.